submit_pm.sh

```diff
@@ -34,6 +32,9 @@ sed "s/.*output_filename.*/        output_filename: ${BENCHY_OUTPUT}.json/" ${BE
 export BENCHY_CONFIG_FILE=benchy-run-${SLURM_JOBID}.yaml
 export MASTER_ADDR=$(hostname)
 
+# Reversing order of GPUs to match default CPU affinities from Slurm
```

Comment:
This is still an issue, no? Or does this setting not work as intended?

Reply:
sorry, nevermind. You added this rather than deleted it. I was misreading things.

Reply:
Yes, it isn't clear to me if this has an impact, but I feel like getting the affinities right is better than not.